### DISCUSSION TITLE
How to avoid LegacyTagJoiner in the logs?

### DISCUSSION THREAD

**Discussion by alrz** [dotnet/extensions#5348] [Q&A]
Using latest version, I see the message is copied in LegacyTagJoiner. Why does this happen and how we can avoid it?

**Comment by RussKie** [comment]
Could you please provide more context for your question?

**Reply by alrz** [reply]
I think this is only related to `HttpLogging` logs where in addition to `message` there's also a `LegacyTagJoiner` field with the same content emitted in the log. I think this condition is related but I'm not sure https://github.com/dotnet/extensions/blob/0cd4e2e66a3b453c15943c3f9c242545775f706c/src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLogger.cs#L42
